src/cli.js

```diff
@@ -0,0 +1,49 @@
+const fs = require('fs');
+const path = require('path')
+
+let inquirer = require('inquirer');
+inquirer
+  .prompt({
+   name: 'ruta',
+   message: '¿Que archivo deseas leer',
+       
+  })
+  .then(answers => {
+      let ruta = answers.ruta;
+    route(ruta)
+   
+  })
+  .catch(error => {
+    if(error.isTtyError) {
+      // Prompt couldn't be rendered in the current environment
+    } else {
+      // Something else when wrong
+    }
+  });
+
+  const route = (rout) =>{
+    let route = require('path').dirname(rout) 
+    cobtenerArchivos(route)
+  }
```

Comment:
Te recomiendo definir mejor tu función y las variables en ingles
```suggestion
  let routeLoader = (path) =>{
    let route = require('path').dirname(path) 
    //getFiles(path)
  }
```